cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts

```diff
@@ -584,6 +605,43 @@ describe('Asset Properties Widget: Configuration/View screen tests', function ()
     cy.deleteCard();
   });
 
+  // Verify file size and file type validations
+  it('TC_Asset_Properties_Widget_view_011', () => {
+    const propKey = 'file';
+    const invalidFileName = 'Image SAG LOGO.png';
+    const invalidFileSize = 'image1.jpg';
+    const errorMessage = 'The selected file is not supported.';
+    const invalidFileSizeErrorMessage= 'The selected file is too large. The size limit is 102.4 kB.'
+    const editPropertyCancelButton = "button[title='Cancel']";
+    const complexPropertyTitle = 'ComplexProperty';
+    cy.selectAssetPropertyAndSave('Test Asset3', propKey);
+    cy.clickPropertyEditButton('File');
+    cy.get('.dlt-c8y-icon-plus-square').click();
+    selectFile('File', invalidFileSize);
+    verifyFileValidationError(invalidFileSizeErrorMessage);
+    selectFile('File', invalidFileName);
+    verifyFileValidationError(errorMessage);
+    cy.get(editPropertyCancelButton).click();
+    cy.get(asset_properties_widget_elements.settingsButton).click();
+    cy.get(asset_properties_widget_elements.editWidgetButton).click();
+    cy.intercept({ method: 'GET', url: '/inventory/managedObjects/**/childAdditions?pageSize=2000&query=%24filter%3D(has(%27c8y_IsAssetProperty%27))' }).as(
```

Comment:
An intercept should be added before making the actual call.

Reply:
@AmarjyotiSAG please have a look at this

Reply:
incorporated.